CHANGELOG.next.asciidoc

```diff
@@ -365,6 +363,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add ELB fileset to AWS module. {pull}14020[14020]
 - Add module for MISP (Malware Information Sharing Platform). {pull}13805[13805]
 - Add `source.bytes` and `source.packets` for uni-directional netflow events. {pull}14111[14111]
+- Add support for gzipped files in S3 input. {pull}13980[13980]
+- Add support for all the ObjectCreated events in S3 input. {pull}14077[14077]
```

Comment:
I think this PR accidentally moves these lines. Must also add a new one for this change

Reply:
I moved these two because they are in the wrong session of the change log 😬